lib/QubitFlatfileImport.class.php

```diff
@@ -1206,7 +1206,7 @@ protected function rowProcessingBeforeSave($row)
         // otherwise, if column is data and a handler for it is set, use it
         call_user_func_array(
           $self->handlers[$columnName],
-          array($self, $value)
+          array(&$self, $value)
```

Comment:
Makes sense.  I have completed the change to remove the references.  Tested on both PHP7 and 5.5.